spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /index' do
+    it 'returns http success' do
+      get '/users/0/posts'
```

Comment:
- [ ] **[OPTIONAL]** I suggest you have a more descriptive path here something like ` get '/users/1/posts'` because `id` start counting from `1`